packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -108,9 +114,25 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         sharedObjectRegistry: ISharedObjectRegistry,
         existing: boolean,
     ): FluidDataStoreRuntime {
-        return new FluidDataStoreRuntime(context, sharedObjectRegistry, existing);
+        return new FluidDataStoreRuntime(
+            context,
+            sharedObjectRegistry,
+            existing,
+            async (rt) => requestFluidObject(rt, "/"));
     }
 
+    /**
+     * Fluid handle to the data store runtime. It'll only be defined if initializeEntrypoint was passed to the
```

Comment:
I'd change that description. There should be no handles to data store runtime. Only to user objects exposed by data store runtime. That difference is important, as "components" implemented on top of data store should not expose internal guts of data store, only their own structure.

Reply:
I think the new description should be more accurate, but double check me on that :)